src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -436,25 +436,15 @@ public override void VisitSwitchCase(ISwitchCaseOperation operation)
             Unindent();
         }
 
-        public override void VisitDoLoop(IDoLoopOperation operation)
-        {
-            LogString(nameof(IDoLoopOperation));
-
-            LogString($" (DoLoopKind: {operation.DoLoopKind})");
-            LogLoopStatementHeader(operation);
-
-            Visit(operation.Condition, "Condition");
-            Visit(operation.IgnoredCondition, "IgnoredCondition");
-            Visit(operation.Body, "Body");
-        }
-
         public override void VisitWhileLoop(IWhileLoopOperation operation)
         {
             LogString(nameof(IWhileLoopOperation));
+            LogString($" (ConditionIsTop: {operation.ConditionIsTop}, ConditionIsUntil: {operation.ConditionIsUntil})");
             LogLoopStatementHeader(operation);
 
             Visit(operation.Condition, "Condition");
             Visit(operation.Body, "Body");
+            Visit(operation.IgnoredCondition, "IgnoredCondition");
```

Comment:
Why not print this as DoLoop did, so we minimize diff?

Reply:
DoLoop also always printed the IgnoreCondition even when null, more diffs are due to the fact that now even while loops have this field - I think we decided on printing null members in the dumper for more clarity some time ago.

Reply:
There are more than 5 diffs of just moving the ignored condition. It's not a big deal though.